src/task/tests/runners/azurePipelinesRunnerInvoker.spec.ts

```diff
@@ -0,0 +1,165 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
```

Comment:
Just wondering if there is any reason why we are using this comment over the "Copyright (C) Microsoft Corporation. All rights reserved." one? Maybe it doesn't matter which one we use?

Reply:
This follows the internal guidance. I'll point you to it offline.